common/models/Question.ts

```diff
@@ -1,4 +1,4 @@
 export interface Question {
-	id: number;
+	id?: number;
```

Comment:
This made more sense, since we can have a non-saved question which has no ID yet.